Detectors/PHOS/simulation/src/GeometryParams.cxx

```diff
@@ -404,7 +390,8 @@ GeometryParams::GeometryParams(const std::string_view name)
     mPHOSAngle[i - 1] = -angle;
   }
 
-  Float_t r = mIPtoOuterCoverDistance + mPHOSParams[3] - mCPVBoxSize[1];
+//  Float_t r = mIPtoOuterCoverDistance + mPHOSParams[3] - mCPVBoxSize[1];
```

Comment:
there are no CPV related variables any more, better delete than comment the line